djautotask/sync_rest.py

```diff
@@ -24,6 +24,12 @@ class Synchronizer:
     lookup_key = 'id'
     last_updated_field = 'lastActivityDate'
 
+    # optional member variables for fetch_records_by_altered_condition method
+    changing_condition_pool = None
+    changing_condition_field_name = None
+    get_page_method_name = None
+    changing_condition_num = 100
```

Comment:
Does this determine the size of the batches? If so, there is already a Constance option for controlling batch size `DJAUTOTASK_BATCH_QUERY_SIZE` and we might be able to use that. If it doesn't conflict with anything then we should use it.